src/app/services/baw-api/baw-api.service.ts

```diff
@@ -199,6 +199,26 @@ export class BawApiService<
     );
   }
 
+  /**
+   * Get a single response from a filter route
+   *
+   * @param classBuilder Model to create
+   * @param path API path
+   * @param filters API filters
+   */
+  public filterShow(
+    classBuilder: ClassBuilder,
+    path: string,
+    filters: Filters<Model>,
+    opts?: NotificationsOpt
+  ): Observable<Model> {
+    return this.session.authTrigger.pipe(
+      switchMap(() => this.httpPost(path, filters)),
+      map(this.handleSingleResponse(classBuilder)),
+      catchError((err) => this.handleError(err, opts?.disableNotification))
+    );
+  }
+
```

Comment:
this make me think my api is a code-smell

something needs to be rethought here

Reply:
We discussed this in the call and it was due to two factors

a. It doesn't implement the service interface (therefore the calling parameters from the resolver are incorrect)
b. (what this comment is addressing) - It's more due to the choice of name. We discussed alternative names such as "filterCreate" for "createFromFilter"

Reply:
I still don't like the name, but you did choose a good name based on the conventions in the file. Leave it as is.